src/nni_manager/core/nnimanager.ts

```diff
@@ -542,9 +544,29 @@ class NNIManager implements Manager {
         this.dispatcher.sendCommand(REPORT_METRIC_DATA, metric.data);
     }
 
+    private requestTrialJobs(jobNum: number): void {
+        if (jobNum < 1) {
```

Comment:
Do you think we need to assert here? or just skip is enough

Reply:
Skip is better here, checking can be done outside if necessary.